CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
    ```
 
-3. Get the code
+3. Get, build and test the code
 
    ```bash
    git clone https://github.com/citusdata/citus.git
+
+   cd citus
+   ./configure
+   make
+   sudo make install
+   cd src/test/regress
+   make check
    ```
 
-4. Build and test
+#### Linux with Apt-Get (Ubuntu, Debian)
```

Comment:
Can we just say _Debian-based Linux_ or something? If not, I'd prefer that `apt-get` be lowercase and monospace.